File: app/src/main/java/com/letianpai/robot/taskservice/TaskServiceApplication.java
package com.letianpai.robot.taskservice;

import android.app.Application;
import com.letianpai.robot.components.utils.GeeUILogUtils;

public class TaskServiceApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        GeeUILogUtils.initXlog2("taskservice", this);
    }
}

File: app/src/main/java/com/letianpai/robot/taskservice/TaskServiceApplication.kt
package com.letianpai.robot.taskservice

import android.app.Application
import com.letianpai.robot.components.utils.GeeUILogUtils

class TaskServiceApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        GeeUILogUtils.initXlog2("taskservice", this)
    }
}
